packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/integration_test/resources/icon_image_base64.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const String iconImageBase64 =
```

Comment:
Same here, Document how to recreate this value if we need to. Also if this needs to be the same as the java version then add a comment saying that as well.